f/connectors/globalforestwatch/gfw_alerts.py

```diff
@@ -193,3 +219,129 @@ def format_alerts_as_geojson(alerts: list, type_of_alert: str):
+    alert_descriptions = {
+        "nasa_viirs_fire_alerts": "fires",
+        "gfw_integrated_alerts": "deforestation",
+        "glad_alerts": "deforestation",
+        "radd_alerts": "deforestation",
+        "modis_alerts": "deforestation",
+    }
+
+    description_alerts = alert_descriptions.get(type_of_alert, "deforestation")
+
+    # Count alerts by month
+    alerts_by_month = {}
+    for alert in alerts:
+        # Extract date from alert (format varies by alert type)
+        if type_of_alert == "nasa_viirs_fire_alerts":
+            date_str = alert.get("alert__date")
+        else:
+            date_str = alert.get(f"{type_of_alert}__date")
+
+        if date_str:
+            # Parse date to get year and month
+            date_parts = date_str.split("-")
+            alert_year = int(date_parts[0])
+            alert_month = int(date_parts[1])
+
+            # Only count alerts within our date range
+            if (
+                alert_year > start_year
+                or (alert_year == start_year and alert_month >= start_month)
+            ) and (
+                alert_year < end_year
+                or (alert_year == end_year and alert_month <= end_month)
+            ):
+                month_key = (alert_year, alert_month)
+                alerts_by_month[month_key] = alerts_by_month.get(month_key, 0) + 1
+
+    # Create metadata records for all months from start to current
+    metadata_records = []
+
+    current_year = start_year
+    current_month = start_month
+
+    while (current_year < end_year) or (
+        current_year == end_year and current_month <= end_month
+    ):
+        # Get alert count for this month
+        month_key = (current_year, current_month)
+        month_alerts = alerts_by_month.get(month_key, 0)
+
+        metadata_record = {
+            "_id": f"{type_of_alert}_{current_year}_{current_month:02d}",
+            "month": current_month,
+            "year": current_year,
+            "total_alerts": month_alerts,
+            "description_alerts": description_alerts,
+            "type_alert": type_of_alert,
+            "data_source": "Global Forest Watch",
+        }
+        metadata_records.append(metadata_record)
+
+        # Move to next month
+        current_month += 1
+        if current_month > 12:
+            current_month = 1
+            current_year += 1
+
+    total_alerts = sum(alerts_by_month.values())
+    logger.info(
+        f"Prepared metadata for {len(metadata_records)} months ({start_year}-{start_month:02d} to {end_year}-{end_month:02d}): {total_alerts} total alerts distributed by month"
+    )
+    return metadata_records
+
+
+def create_gfw_metadata_table(cursor, table_name):
+    """Create the metadata table for GFW alerts."""
+    cursor.execute(
+        sql.SQL("""
+        CREATE TABLE IF NOT EXISTS {metadata_table} (
+            _id character varying(100) NOT NULL PRIMARY KEY,
+            month bigint,
+            year bigint,
```

Comment:
In your PR description you say that these are updated _daily_ but we are only tracking updates at a monthly resolution? Why?

Reply:
Also: there's really no need for `month` and `year` to be `bigint`. `SMALLINT` would be fine for these types of vals.